_includes/footer.html

```diff
@@ -2,58 +2,55 @@
+         <li><a href="/services/keycloak">AeroGear <strong>Keycloak</strong></a></li>
         </ul>
       </div><!-- col -->
-      
+
       <div class="col-sm-3">
-        <h5>Aerogear</h5>
+        <h5>Mobile SDKs</h5>
         <ul class="list-unstyled">
-          <li><a href="/getstarted/downloads">Downloads and snippets</a></li>
-          <li><a href="/getstarted/guides">Guides</a></li>
-          <li><a href="/getstarted/demos">Demos</a></li>
-          <li><a href="/docs/specs/">Docs</a></li>
-          <li><a href="/docs/planning/">Road Map</a></li>
+         <li><a href="/sdks/android/">Android</a></li>
+         <li><a href="/sdks/cordova/">Cordova</li>
+         <li><a href="/sdks/ios/">iOS</a></li>
+         <li><a href="/sdks/xamarin/">Xamarin</a></li>
         </ul>
       </div><!-- col -->
       
-      
       <div class="col-sm-3">
         <h5>COMMUNITY</h5>
         <ul class="list-unstyled">
-          <li><a href="https://github.com/aerogear/">GitHub</a></li>
-          <li><a href="/docs/planning/">Jira Repo Issues</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">Developer mailing list</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-users">Users mailing list</a></li>
+          <li><a target="_blank" href="https://github.com/aerogear/">GitHub - Aerogear</a></li>
+          <li><a target="_blank" href="https://github.com/aerogearcatalog/">GitHub - Aerogear Catalog</a></li>
+          <li><a target="_blank" href="/docs/planning/">Jira Repo Issues</a></li>
+          <li><a  href="/community/contribution-guide/">Contribution Guide</a></li>
+          <li><a target="_blank" href="https://groups.google.com/forum/#!forum/aerogear">Google Group</a></li>
           <li><a href="irc://irc.freenode.net/aerogear">IRC</a></li>
         </ul>
       </div><!-- col -->
 
-
-
       <div class="col-sm-12 follow">
         <h5>FOLLOW US</h5>
         <ul class="list-inline">
-          <li><a href="https://www.twitter.com/aerogears"><i class="fa fa-twitter-square"></i> Twitter</a></li>
-          <li><a href="https://www.facebook.com/AeroGearProject"><i class="fa fa-facebook-square"></i> Facebook</a></li>
-          <li><a href="https://plus.google.com/106588661437153585722/posts"><i class="fa fa-google-plus-square"></i> Google+</a></li>
-          <li><a href="/feed.atom"><i class="fa fa-rss-square"></i> Feed</a></li>
+          <li><a target="_blank" href="https://www.twitter.com/aerogears"><i class="fa fa-twitter-square"></i> Twitter</a></li>
+          <li><a target="_blank" href="https://www.facebook.com/AeroGearProject"><i class="fa fa-facebook-square"></i> Facebook</a></li>
+          <li><a target="_blank" href="https://plus.google.com/106588661437153585722/posts"><i class="fa fa-google-plus-square"></i> Google+</a></li>
+          <li><a target="_blank" href="/feed.atom"><i class="fa fa-rss-square"></i> Feed</a></li>
```

Comment:
This one seems to become invisible from time to time and then shows when it's hovered over.

Reply:
it it's a general css issue for all links located at the bottom of the page - will file a ticket for that